### PR TITLE
feat: Flexible side pool

### PR DESCRIPTION
**Pull request by memoyil**

### CODE REVIEW

**Comment by ChefMomota** [comment]
Screenshot 2022-06-08 at 12 55 41 PM

**Comment by memoyil** [comment]
> First at /pools page after that go to home page
> 
> Screenshot 2022-06-08 at 12 55 41 PM

Done

File: src/views/Pools/index.tsx [hunk 1]
```diff
@@ -118,6 +118,19 @@ const sortPools = (account: string, sortOption: string, pools: DeserializedPool[
...
+            const vault = pool as DeserializedPoolVault
+            if (!vault.userData || !vault.userData.userShares) {
+              return 0
+            }
+            return getCakeVaultEarnings(
+              account,
+              vault.userData.cakeAtLastUserAction,
+              vault.userData.userShares,
+              vault.pricePerFullShare,
+              vault.earningTokenPrice,
```
Comment (chef-jojo) [inline src/views/Pools/index.tsx:131]: how about `pool.vaultKey === VaultKey.CakeFlexibleSideVault ? undefined : vault.userData.currentOverdueFee.plus(vault.userData.currentPerformanceFee)`

Comment (memoyil) [reply src/views/Pools/index.tsx:131]: Done

**Comment by chef-jojo** [comment]
total staked should be both cake pool and side cake pool sum

**Comment by memoyil** [comment]
> total staked should be both cake pool and side cake pool sum

Total staked at which pool you mean? Because total staked in cake pool includes side cool pool too

**Comment by chef-jojo** [comment]
migration page broken, i think userShare checking is needed?

**Review by chef-jojo — APPROVED** [review]

**Pull request merged**